core/src/main/java/org/mskcc/cbio/oncokb/util/AlterationUtils.java

```diff
@@ -459,6 +459,17 @@ public static Alteration getTruncatingMutations(Gene gene) {
         return alterations;
     }
 
+    public static Set<Alteration> getVUS(Alteration alteration){
+        Set<Alteration> result = new HashSet<>();
+        Gene gene = alteration.getGene();
+        if (CacheUtils.isEnabled()) {
+            result = CacheUtils.getVUS(gene.getEntrezGeneId());
+        } else {
+            result = AlterationUtils.findVUSFromEvidences(EvidenceUtils.getEvidenceByGenes(Collections.singleton(gene)).get(gene));
+        }
```

Comment:
Let's move this logic to a separate method so `excludeVUS` can use it too?

Reply:
I think the total `getVUS()` can be used in `excludeVUS()`